pkg/clustersync/clusterSync_test.go

```diff
@@ -217,9 +217,10 @@ func Test_ProcessClusterDeleteOnMC(t *testing.T) {
 
 }
 
-func Test_ProcessClusterDeleteOnMCA(t *testing.T) {
+//Delete only if addon name is search-collector
```

Comment:
Would it be easy to add a test to validate that other addons are correctly filtered out?

Reply:
That will require informer tests - will need more work. Will add in another PR.